### PR TITLE
Adding prom_aggregation_gateway

### PR DESCRIPTION
**Pull request by rohit-dimagi**

### CODE REVIEW

File: roles/agg_pushgateway_exporter/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+prom_aggregation_gateway_install_dir: "/opt/prometheus-exporters/prom-aggregation-gateway"
+prom_aggregation_gateway_binary: "{{ prom_aggregation_gateway_install_dir }}/prom-aggregation-gateway"
+prom_aggregation_gateway_install_url: "github.com/LarkTechnologies/prom-aggregation-gateway"
```
Comment (snopoke) [inline roles/agg_pushgateway_exporter/defaults/main.yml]: you'll need to use fork and branch: https://github.com/dimagi/prom-aggregation-gateway/tree/sk/guage-aggregation

Comment (rohit-dimagi) [reply roles/agg_pushgateway_exporter/defaults/main.yml]: Changed.

**Comment by rohit-dimagi** [comment]
@snopoke  will this go on every celery machine ?

**Comment by snopoke** [comment]
No, we just need a single instance of this service.

**Pull request merged**